<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rfidAppClass</class>
 <widget class="QMainWindow" name="rfidAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>rfidApp</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnections">
    <property name="title">
     <string>Connections</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <widget class="QMenu" name="menuScan">
    <property name="title">
     <string>Scan</string>
    </property>
   </widget>
   <addaction name="menuConnections"/>
   <addaction name="menuScan"/>
  </widget>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionQuit_now">
   <property name="text">
    <string>Quit now</string>
   </property>
  </action>
  <action name="actionScan_for_Tags">
   <property name="text">
    <string>Scan for Tags</string>
   </property>
  </action>
  <action name="actionOn">
   <property name="text">
    <string>On</string>
   </property>
  </action>
  <action name="actionOff">
   <property name="text">
    <string>Off</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rfidApp.qrc"/>
 </resources>
 <connections/>
</ui>
